app/views/parties/new.html.erb

```diff
@@ -1,18 +1,24 @@
 <h1>Viewing Party Details</h1>
 
-<%= form_with model: @party, url: parties_path, local: true do |f| %>
-  <span>Movie Title:</span>
-  <%= @movie[:title] %><br>
+<h2><%= "Viewing party for: #{@movie[:title]}" %></h2>
 
+<%= form_with model: @party, url: parties_path, local: true do |f| %>
   <%= f.label :duration %>
   <%= f.number_field :duration, value: @movie[:runtime] %><br>
 
   <%= f.label :party_date %>
-  <%= f.date_field :party_date, min: Date.today%><br>
+  <%= f.date_select :party_date, min: Date.today, include_blank: true, defaul: nil, start_year: Date.current.year, order: [:day, :month, :year] %><br>
```

Comment:
Dates were coming in as month, day, year but Ruby didn't know how to convert to date (kept doing day, month, year)